src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -49,20 +45,20 @@ public static void main(String[] args) {
      * @param client - a {@link HazelcastInstance} client.
      */
     private static void mapExample(HazelcastInstance client) {
-        System.out.println("Now the map named 'map' will be filled with random entries.");
-
-        IMap<String, String> map = client.getMap("map");
-        Random random = new Random();
-        int iterationCounter = 0;
-        while (true) {
-            int randomKey = random.nextInt(100_000);
-            map.put("key-" + randomKey, "value-" + randomKey);
-            map.get("key-" + random.nextInt(100_000));
-            if (++iterationCounter == 10) {
-                iterationCounter = 0;
-                System.out.println("Current map size: " + map.size());
-            }
-        }
+        IMap<String, HazelcastJsonValue> cities = client.getMap("cities");
+        cities.put("1", City.asJson("United Kingdom", "London", 9_540_576));
+        cities.put("2", City.asJson("United Kingdom", "Manchester", 2_770_434));
+        cities.put("3", City.asJson("United States", "New York", 19_223_191));
+        cities.put("4", City.asJson("United States", "Los Angeles", 3_985_520));
+        cities.put("5", City.asJson("Turkey", "Ankara", 5_309_690));
+        cities.put("6", City.asJson("Turkey", "Istanbul", 15_636_243));
+        cities.put("7", City.asJson("Brazil", "Sao Paulo", 22_429_800));
+        cities.put("8", City.asJson("Brazil", "Rio de Janeiro", 13_634_274));
+
+        int mapSize = cities.size();
+        System.out.printf("'cities' map now contains %d entries.\n", mapSize);
+
+        System.out.println("--------------------");
```

Comment:
Can you please run this sample against empty cluster and see if there is any visible change on the map metrics available in the console. One of the reasons for continuous, looped map put was to be able to observe cluster metrics straight after connecting the client for the first time - therefore exploring some of the hazecast console features. 
I'm afraid that these eight entries might be hardly visible. 

I'm not against the change - providing meaningful data instead of randomized keys makes a lot of sense. But I don't will not to overlook other aspects of the code samples.

Reply:
It results in this kind of graphs
![image](https://user-images.githubusercontent.com/29387993/179983645-d1436421-f20d-400a-919c-3ecacb36022a.png)

The reason Jake wanted to have such a sample is to build a tutorial on top of it. So, we decided on a finite, queryable, small entry set.

Reply:
Well, if you want more graphics you can still do a finite set but a larger set of data, would that solve the problems?

Reply:
We'll be looking at our samples more closely after launch. For now, I don't think it's an issue.